api/client/cluster/objectstore.go

```diff
@@ -0,0 +1,51 @@
+package cluster
+
+import (
+	"github.com/libopenstorage/openstorage/objectstore"
+	"strconv"
+)
+
+const (
+	ObjectStorePath = "/objectstore"
+)
+
+func (c *clusterClient) ObjectStoreInspect() (*objectstore.ObjectstoreInfo, error) {
+	objectstoreInfo := &objectstore.ObjectstoreInfo{}
+	request := c.c.Get().Resource(clusterPath + ObjectStorePath)
+	if err := request.Do().Unmarshal(objectstoreInfo); err != nil {
+		return nil, err
+	}
+	return objectstoreInfo, nil
+}
+
+func (c *clusterClient) ObjectStoreCreate(volume string) error {
+	req := c.c.Post().Resource(clusterPath + ObjectStorePath)
+	req.QueryOption(objectstore.VolumeName, volume)
```

Comment:
@lpabon , I remember you advice on not to use "querystring", but our rest client causes /path variable to automatically convert into lower case characters. I don't know whether this behaviour is expected or not, but volume name can be case sensitive so keeping this as query parameter instead of including in path.

Reply:
great point. Thanks for the explanation. Please add this as a comment to the code.

Reply:
sure, thanks